x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/saved_object.ts

```diff
@@ -14,7 +14,7 @@ import { map, fold } from 'fp-ts/lib/Either';
 import { identity } from 'fp-ts/lib/function';
 
 import type { SavedObjectsFindOptions } from '@kbn/core/server';
-import type { AuthenticatedUser } from '@kbn/security-plugin/common/model';
+import { getUserDisplayName, type AuthenticatedUser } from '@kbn/security-plugin/common/model';
```

Comment:
might make sense to delete this function all together and instead use https://github.com/elastic/kibana/blob/main/packages/kbn-user-profile-components/src/user_profile.ts#L158

Reply:
@kqualters-elastic I haven't deleted the function, it is used internally in the Security (not Security Solution) plugin, I won't do so many changes there.
I changed our import to the `user-profile-components`.